### ISSUE TITLE
Default Learning_Rate

### ISSUE THREAD

**Issue by neonadia** [cmu-db/ottertune#216]
I found there is a default learning rate setting at 0.01 in "gp_tf.py".
Since this is a Hyperparameter, to improve the performance or convergence speed do I need to change this number?

**Comment by dvanaken** [comment]
We've found 0.01 to be a good value for the learning rate in our experiments but you can test other values to verify this is also true for your data.
